src/docs/admin/references/SchedulerPropertiesReference.adoc

```diff
@@ -163,6 +163,9 @@ pa.scheduler.job.logs.location=logs/jobs/
 # Size limit for job and task logs in bytes
 pa.scheduler.job.logs.max.size=10000
 
+# Logger format pattern
+pa.scheduler.job.logs.pattern=[%X{job.id}t%X{task.id}@%X{host};%d{HH:mm:ss}] %m %n
```

Comment:
better to specify that this is the task output log. 
The property could be called 
pa.scheduler.job.task.output.logs.pattern